gitlabform/lists/projects.py

```diff
@@ -112,6 +99,36 @@ def _get_projects(self, target: str, groups: Groups) -> Projects:
                 OmissionReason.ARCHIVED,
                 archived_projects_from_configuration_not_from_groups,
             )
+        else:
+            # in all other cases, we also need to look for projects in the config
+            # that are being transferred to a different namespace
+            for project in projects_from_configuration_not_from_groups:
+                # if the target is a group, and the project is not in the group
+                if (
+                    len(groups.get_effective()) == 1
+                    and target != project.rsplit("/", 1)[0]
+                ):
+                    debug(
+                        "Ignore project '%s', since it's not in target group '%s",
+                        project,
+                        target,
+                    )
+                else:
+                    project_transfer_source = (
+                        self._get_project_transfer_source_from_config(project)
+                    )
+
+                    if project_transfer_source:
+                        if self._find_project_transfer_source_in_gitlab(
+                            project_transfer_source
+                        ):
+                            projects.add_requested([project])
+                        else:
+                            fatal(
+                                f"""Configuration contains project {project} to be transferred from {project_transfer_source}
+                                    but the source project cannot be found in GitLab!""",
+                                exit_code=EXIT_INVALID_INPUT,
+                            )
```

Comment:
I think we can simplify the code using the walrus operator mentioned above.

Reply:
Updated.

Reply:
Thanks for the update.